### ISSUE TITLE
Widget: show only a particular calendar

### ISSUE THREAD

**Issue by andreapx** [Etar-Group/Etar-Calendar#1575]
It would be nice to be able to show only a particular calendar in the widget.
Is this something that is already possible and I've missed?
Thanks!

**Comment by liz-desartiges** [comment]
I agree, one thing missing is the ability to configure what calendar a(/the) widget displays. I have access to calendar of relatives and on my home screen I only want my stuff to be displayed
